lib/ts/framework/utils.ts

```diff
@@ -138,7 +138,7 @@ export async function assertThatBodyParserHasBeenUsedForExpressLikeRequest(
             let err = await new Promise((resolve) => {
                 let resolvedCalled = false;
                 if (request.readable) {
-                    jsonParser(request, new ServerResponse(request), (e) => {
+                    jsonParser(request, new ServerResponse(request), (e: any) => {
```

Comment:
can we not use `any` type?

Reply:
you can use, i did any as i don't know what types are